docs/kubernetes.md

```diff
@@ -32,6 +32,34 @@ In the image above, you can see the following parts:
 
 All VMs are in the same private VNET and are fully accessible to each other.
 
+
+## Optional: Enable network policy enforcement using Calico

```

Comment:
Noted, the docs need some attention, I'll keep this in mind. The cross reference is a good solution.